examples/kv_events/README.md

```diff
@@ -21,16 +21,16 @@ make download-tokenizer
 go run -ldflags="-extldflags '-L$(pwd)/lib'" examples/kv_events/offline/main.go
 ```
 
-The example will start the KV-Cache Manager (indexer) and a dummy publisher that simulates KV-Events. 
+The example will start the KV-Cache (indexer) and a dummy publisher that simulates KV-Events. 
```

Comment:
Parenthesis around `indexer` should be removed.

Same applies to the instances in the list below.